### DISCUSSION TITLE
What is the current state of SKR Pro 1.x boards and TFT?

### DISCUSSION THREAD

**Discussion by raphael303** [grblHAL/core#305] [General]
Hi

What is the current state of using GRBLHal with SKR pro 1.x boards?
How about the TFT?

I‘m using an MPCNC with the v1-adapted marlin firmware and cnjs right now. However, since laser is supposed to be better with GRBL I consider switching.

However, it depends the hassle I am facing with such a switch. Also the TFT still working would be nice.

**Comment by terjeio** [comment]
> What is the current state of using GRBLHal with SKR pro 1.x boards?

I guess the state is "working" - I do not have a board myself so cannot verify, but AFAIK there are users out there.

> How about the TFT?

Not supported, somebody could write a plugin to support TFT UART comms? But the firmware in the display does not support CNC (Grbl) mode, only 3D printers?
Driving the display over SPI is possible, again if someone write a plugin for that. But perhaps it will be easier to reprogram the display MCU to support the grblHAL MPG interface?
